ginga/misc/plugins/Histogram.py

```diff
@@ -441,10 +444,26 @@ def cut_levels(self):
         try:
             loval = float(self.w.cut_low.get_text())
             hival = float(self.w.cut_high.get_text())
+        except Exception as e:
+            errmsg = 'Error cutting levels: {0}'.format(str(e))
+            self.fv.showStatus(errmsg)
+            self.logger.error(errmsg)
+            return
 
+        # This is needed to avoid UserWarning from matplotlib
+        if loval == hival:
```

Comment:
True, that.  Well, I guess worst case is it doesn't catch the equivalence and we get the same error as before.  It might be worth looking into something like `np.isclose()`.  The only problem is that for cut levels the values *can* be very, very close.